go.mod

```diff
@@ -5,86 +5,90 @@ go 1.18
 require (
 	github.com/go-logr/logr v1.2.3
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.18.1
+	github.com/onsi/gomega v1.20.0
 	github.com/openshift/api v3.9.0+incompatible
-	github.com/openstack-k8s-operators/keystone-operator v0.0.0-20220727152756-f13e087e4199
-	github.com/openstack-k8s-operators/keystone-operator/api v0.0.0-20220727152756-f13e087e4199
-	github.com/openstack-k8s-operators/lib-common v0.0.0-20220727151739-0ac9fed768b3
-	github.com/openstack-k8s-operators/mariadb-operator v0.0.0-20220516121356-119f8d825a71
-	k8s.io/api v0.24.2
-	k8s.io/apimachinery v0.24.2
-	k8s.io/client-go v0.24.2
-	sigs.k8s.io/controller-runtime v0.12.1
+	github.com/openstack-k8s-operators/keystone-operator v0.0.0-20220810095541-da254cc1440b
+	github.com/openstack-k8s-operators/keystone-operator/api v0.0.0-20220810095541-da254cc1440b
+	github.com/openstack-k8s-operators/lib-common/modules/common v0.0.0-20220810140921-24936441aa9b
+	github.com/openstack-k8s-operators/lib-common/modules/database v0.0.0-20220809134547-1a7f2be42190
```

Comment:
probably good to keep those in sync and the one in /api

Reply:
Ack